dev/fattest.simplicity/src/componenttest/topology/utils/tck/TCKUtilities.java

```
@@ -865,4 +874,81 @@ static void assertServerHappy(LibertyServer server) throws Exception {
         assertThat("Server log has errors after starting", server.findStringsInLogs("CWWKF0002E"), empty()); // A bundle could not be found for ...
         assertThat("Server log has errors after starting", server.findStringsInLogs("CWWKE0702E"), empty()); // Could not resolve module ...
     }
+
+    private static Set<String> detectedShortNames = null;
+
+    private static Set<String> getDetectedShortNames() {
+        if (detectedShortNames != null) {
+            return detectedShortNames;
+        }
+
+        detectedShortNames = new HashSet<String>();
+        Bootstrap b;
+        try {
+            b = Bootstrap.getInstance();
+        } catch (Exception e) {
+            Log.info(c, "getDetectedShortNames", "Bootstrap.getInstance() failed. This will likely cause the current repeat to skip " + e.toString());
+            detectedShortNames = null; //if bootstrap is broken we've got bigger problems, but we can avoid caching an exception
+            return new HashSet<String>();
+        }
```

I agree with avoiding caching a bad value, but I think we should throw the exception so that we know why the test failed.

With the current code, we might skip the repeat and not realise that nothing ran.

Also in the other catch block.